### PR TITLE
fix(tutorial): fix first start of tutorial

### PR DESCRIPTION
**Pull request by christianalfoni**

### CODE REVIEW

**Comment by fopsdev** [comment]
would like to have better news but...
tried this pull, now browser starts but doesn't refresh on changes or chapter changes by inquirer  (both: linux and win)

**Comment by christianalfoni** [comment]
@fopsdev Yeah, that is where we are at... cause it works when you do: `npm run tutorial`, right?

**Comment by christianalfoni** [comment]
@fopsdev Could you try replacing the code in `prepare.js` with this?

```js
/* global rm, cp, sed */
var path = require('path')
var paths = require('./paths')

function prepare (part) {
  cp(path.join(paths.parts, part, 'src', 'index.js'), paths.src)
  cp('-R', path.join(paths.parts, part, 'src', 'components'), paths.src)
  cp(paths.partsIndex, paths.public)
  sed('-i', '%N%', part, paths.publicIndex)
}

module.exports = prepare

```

**Comment by fopsdev** [comment]
btw. tried things like:
```
function prepare (part) {
...
... 
 touch('-a',paths.root + '/src/index.js')
 touch('-a',paths.root + '/build/bundle.js')

}
```

and

watchOptions: {
  poll: true
},
inside webpack.config.dev

(https://github.com/webpack/webpack-dev-server/issues/155)

but at no avail...

whish you guys luck hunting this one down. i'm off to prague this weekend :)

**Comment by christianalfoni** [comment]
@fopsdev Did you try my suggestion? :)

**Comment by fopsdev** [comment]
oh didn't see your proposal, will try it right away :)

**Comment by christianalfoni** [comment]
@fopsdev And have fun i Prague! :-D

**Comment by fopsdev** [comment]
wow that worked :), now getting a compile error when selecting 11...lemme see

**Comment by christianalfoni** [comment]
Ah, sweet :) I will change to that, awaiting your response on chapter 11 issue :)

**Comment by christianalfoni** [comment]
Ah, I see, issue... will fix it. You pack your bags and have fun! :D

**Comment by fopsdev** [comment]
haha, will do, will drink a tank beer for you guys! :)

**Comment by christianalfoni** [comment]
Well deserved! ;-)

**Comment by christianalfoni** [comment]
Okay, I think this one can be brought in for more testing @guria. Now it never delete index.js, only copies over. That is the issue

File: packages/tutorial/scripts/prepare.js [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
+  }
+
+  ls(path.join(paths.parts, part, 'src')).forEach(function (file) {
+    cp('-R', path.join(paths.parts, part, 'src', file), paths.src)
+  })
   cp(paths.partsIndex, paths.public)
   sed('-i', '%N%', part, paths.publicIndex)
 }
 
-if (process.argv.length > 2 && process.argv[2].indexOf('prepare.js') >= 0) {
```
Comment (Guria) [inline packages/tutorial/scripts/prepare.js]: @christianalfoni why this removed? I see it still used in package.json preapare and chapter scripts. Not sure if it should be cleaned up or it still can be used.

Comment (christianalfoni) [inline packages/tutorial/scripts/prepare.js]: Oh, let me check that

Comment (christianalfoni) [inline packages/tutorial/scripts/prepare.js]: Okay, I removed package.json scripts. It is only used as a function now trough inquirer :)

**Pull request merged**